content/getting-started/building-an-ionic-app.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Building an Ionic app 
+description: Building an Ionic app with YAML.
+weight: 7
+aliases:
+  - '../yaml/building-an-ionic-app'
+---
+
+## Setting up an Ionic Capacitor/Cordova project
+
+The apps you have available on Codemagic are listed on the Applications page. See how to add additional apps [here](./adding-apps-from-custom-sources).
+
+1. On the Applications page, click **Set up build** next to the app you want to start building. 
+2. Select **Ionic App** as the starter workflow.
+3. Download the example configuration for Ionic App or copy it to clipboard.
+4. Then edit the configuration file to adjust it to your project needs and commit it to the root of your repository.
+    * For an overview about using `codemagic.yaml`, please refer [here](./yaml). 
+    * Read more about adding configuration for [testing](../testing-yaml/testing), [code signing](../code-signing-yaml/signing) and [publishing](../publishing-yaml/distribution).
+    * See the full Android workflow example [below](#android-workflow-example).
```

Comment:
Should be something along the lines of "See the full workflow examples below" and no link.